### DISCUSSION TITLE
skypeforlinux and zoom don't operate with the OBS virtual camera

### DISCUSSION THREAD

**Discussion by parametalol** [umlaeute/v4l2loopback#586] [Q&A]
Hello,

I'm not sure where the actual problem is, but on my

* Fedora 40 beta
* `v4l2loopback.x86_64` **0.13.1-1.fc40** from @rpmfusion-free;
* kernel 6.8.5-301;
* `options v4l2loopback video_nr=10 card_label="OBS Camera" exclusive_caps=1` in modprobe conf.

I experience the following:

* OBS 30.1.2 shows no problems, virtual camera starts fine;
* Google Meet in Vivaldi browser finds the OBS Camera and works fine;
* Zoom (installed as rpm) doesn't see the virtual camera at all;
* skypeforlinux insider does see the camera, but shows only black.
* the Gnome Camera (snapshot) only shows the physical camera (well, I'm not even sure it knows how to manage cameras).

Any clues if any of those apps are possible to fix somehow?

Thanks!

**Comment by umlaeute** [comment]
i cannot test with skype4linux, but here with
- Debian/sid 
- linux 6.7.9-amd64
- v4l2loopback v13.1
- OBS 30.0.2.1
- zoom 6.0.0 (4563)

everything works as expected

**Comment by parametalol** [comment]
Works now, after don't know what happened!
